openedx_lti_tool_plugin/models.py

```diff
@@ -124,3 +129,74 @@ def save(self, *args: tuple, **kwargs: dict):
     def __str__(self) -> str:
         """Get a string representation of this model instance."""
         return f'<LtiProfile, ID: {self.id}>'
+
+
+class CourseAccessConfiguration(models.Model):
+    """Course access configuration.
+
+    A model to store LTI tool course access configuration.
+    """
+
+    EXAMPLE_ID_LIST = 'Example: ["id-1", "id-2", ...]'
+
+    lti_tool = models.OneToOneField(
+        LtiTool,
+        on_delete=models.CASCADE,
+        related_name='openedx_lti_tool_plugin_course_permission',
+        verbose_name=_('LTI Tool'),
+    )
+    allowed_course_ids = models.TextField(
+        verbose_name=_('Allowed Course IDs'),
+        help_text=_(f'List of allowed courses IDs. {EXAMPLE_ID_LIST}'),
+        default=[],
+    )
```

Comment:
Is this field going to be used only trough the admin panel?. If yes, what do you think about including a course id validator?.
Didn't you consider making a many to many relationship to courses? I suppose this is not going to be friendly in admin panel.

@kuipumu